web/src/games/bullsAndCows/locales/pt.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Touros & Vacas",
+  "description": "parecido com Senha",
+  "descriptionTag": "Jogue Touros e Vacas (também conhecido como Senha) de graça online.",
+  "instructions": {
+    "videoId": "GG7acm844UE",
```

Comment:
On my tests it seems that the video still showing the original (english one), did I miss something?

Reply:
I am not sure, I can try to debug this weekend

Reply:
So, after we got a cease and desist from another game, we don't want to have any reference in videos to other (copyrighted) games. The only exception is "similar to X" which is protected by copyright law.

This video has references to the other game, so can you please remove the video?

Reply:
It's working fine EN/PT :D